dist/styles.css

```diff
@@ -1,15 +1,7 @@
 body{
-  	font-family: 'Exo 2', sans-serif;
+  font-family: 'Exo 2', sans-serif;
```

Comment:
Can you explain why do we need to fix the indentation, it seems fine for me. Infact, its not following the indentation after your change.
![](https://user-images.githubusercontent.com/2767425/36882087-d5e0636e-1df7-11e8-9db8-f4739387066d.png)